officemanager/department/migrations/0003_auto_20190904_1324.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 2.2.4 on 2019-09-04 11:06
+
+from django.db import migrations
+from random import randint, random
+
+def add_employees(apps, schema_editor):
```

Comment:
There is a more efficient way for inserting multiple records into a table - bulk_create: https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-create